build_all.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
So this file is for the one off creation that we could say, store in Docker Hub?  My only concern is that a local user misinterprets this script and runs it locally....

Reply:
I understand your concern, there are two ways to resolve that.
1. Running it locally is ok as the push commands will fail unless you have the right login credentials for the official repo.
2. Move this script to Jenkins
I'd vote for 1. as it is easy to maintain the script.

We can add a WARNING message with a sleep incase the script is invoked by mistake.

Reply:
Also, I have one more such script (update_manifest_all.sh), which I will submit in a subsequent PR, that will create manifest commands for all versions and push to hub.docker.com. Having it on github will help easy review and maintenance. Another idea would be to move these two scripts to a separate dir, say docker-library.